test/spec/modules/sovrnBidAdapter_spec.js

```diff
@@ -73,7 +76,7 @@ describe('sovrnBidAdapter', function() {
       expect(payload.imp[0].banner.h).to.equal(1)
     })
 
-    it('accepts a single array as a size', function() {
+    it('accepts a single array as a size', () => {
```

Comment:
Can you please change this back to `function() {` here (and in other parts of the test file)?  We want to keep it this style to better conform for Mocha.